esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -1046,12 +1047,16 @@ void gatherQuerySetQueryDetails(IEspContext &context, IPropertyTree *query, IEsp
     if (query->hasProp("@warnTimeLimit"))
         queryInfo->setWarnTimeLimit(query->getPropInt("@warnTimeLimit"));
     if (query->hasProp("@priority"))
-        queryInfo->setPriority(getQueryPriorityName(query->getPropInt("@priority")));
```

Comment:
Since this code is in 3 places could write a simple function template to do this?